javascript/demo/demo-node.js

```diff
@@ -1,20 +1,89 @@
-const one = require('oneseismic')
-const http = require('http')
+const one = require("oneseismic");
+const http = require("http");
 
-one().then(mod => {
-    http.get("http://host:port/result/<pid>/stream", {
+/**
+ * To run this demo:
+ *   1. Build the oneseismic library in module mode (aka ONESEISMIC_MODULARIZE=ON)
+ *   2. Let node environment know where to find it (export NODE_PATH=path_to_dir)
+ *   3. Update server, container_id and sas_token.
+ */
+
+one().then((mod) => {
+  server = "<protocol>://<host>:<port>";
+  container_id = "<container_id>";
+  sas_token = "<sas_token>";
+
+  url = `${server}/graphql?${sas_token}`;
```

Comment:
Do we support sending the sas token as an url parameter? I don't think so, and even if we do I would opt for using the authorization header in this demo